apps/prairielearn/elements/pl-image-capture/pl-image-capture.mustache

```diff
@@ -0,0 +1,128 @@
+<script>
+$(function() {
+    new window.PLImageCapture(
+      "{{uuid}}",
+      "{{answer_name}}",
+      "{{external_image_capture_url}}",
+      "{{variant_id}}",
+      "{{submitted_file_name}}",
+      "{{submission_date}}",
+      "{{editable}}",
+      "{{mobile_capture_enabled}}",
+    );
```

Comment:
Nit, some other elements construct a full JSON object in Python:

https://github.com/PrairieLearn/PrairieLearn/blob/09b9386f2e32621310fb2f730435d555dec39870/apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.py#L161

And then interpolate it into the resulting HTML:

https://github.com/PrairieLearn/PrairieLearn/blob/09b9386f2e32621310fb2f730435d555dec39870/apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.mustache#L22

IMO this is a little cleaner.

Reply:
To add to this: one advantage of using JSON as a data attribute is that you can more easily handle non-string options, as well as makes it easier to handle options that may have `<>&'"` as part of their content.